### PR TITLE
Vol add border interior

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Add a small option  to add border inside the volume

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
     (Roland Denis, [#359](https://github.com/DGtal-team/DGtalTools/pull/359/files))
   - Using SourceForge to download doxygen sources during Travis CI jobs.
     (Roland Denis [#360](https://github.com/DGtal-team/DGtalTools/pull/360))
+  - volAddBorder: add an option to fill the border of interior volume (Bertrand
```
Comment (dcoeurjo) [inline ChangeLog.md:14]: I don't understand what it means..

Comment (kerautret) [reply ChangeLog.md:14]: in other words, it changes the image values of voxels located on the frontier of the domain.

Comment (dcoeurjo) [reply ChangeLog.md:14]: Is that :

Add an option that sets zero value to domain boundary voxels without changing the domain extent 

?

Comment (kerautret) [reply ChangeLog.md:14]: yes exactly, thanks I add sorry for the not clear description

File: volumetric/volAddBorder.cpp [hunk 2]
```diff
@@ -102,6 +102,7 @@ int main(int argc, char**argv)
   general_opt.add_options()
     ( "help,h", "display this message." )
     ( "input,i", po::value<std::string>(), "Input vol file." )
+    ( "inside", "Add border inside the volume (set 0 inside the border)." )
```
Comment (dcoeurjo) [inline volumetric/volAddBorder.cpp:105]: Update the option doc as well ?

Comment (kerautret) [reply volumetric/volAddBorder.cpp:105]: yes oops..

Comment (kerautret) [reply volumetric/volAddBorder.cpp]: I also update the doxygen description that I forget ;)

File: ChangeLog.md (hunk 1 above)
Comment (kerautret) [reply ChangeLog.md]: thanks

**Comment by dcoeurjo** [comment]
Thx !;)

**Pull request merged**